controllers/ControllerAdmin.php

```diff
@@ -3,201 +3,212 @@
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 			exit();
 		}
 
 		if ($params['password'] === $params['confirme']){
-			$confirm = true;
-			$this->_userManager->updateUserPass($params, $key);
-			echo $this->_twig->render('profile.twig', ['confirm' => $confirm,
-			                                           'server' => $this->_server,
-			                                           'logged'=> TRUE,
-			                                           'user'=> $key['pseudo'],
-			                                           'value'=> $key['email'],
-			                                           'avatar'=> $key['avatar']]);
+
+			(new AuthManager)->updateUserPass($params, $key);
+
+			echo $this->getTwig()->render('profile.twig', ['confirm' => TRUE,
+			                                               'server' => $this->getServer(),
+			                                               'logged'=> TRUE,
+			                                               'user'=> $key['pseudo'],
+			                                               'value'=> $key['email'],
+			                                               'admin' => $key['admin'],
+			                                               'avatar'=> $key['avatar']]);
 
 		}
 
 	}
 
-	public function checkForm($params){
+
+	/**
+	 * Vérifie les entrées d'utilisateur des formulaires
+	 *
+	 * @param array $params
+	 */
+	public function checkForm(array $params){
 		foreach ($params as $key => $value){
 
 			if ($value != preg_replace('/\s+/', '', $value)){
-				$space = true;
-				echo $this->_twig->render('profile.twig', ['space' => $space,
-				                                           'server' => $this->_server,
-				                                           'logged'=> TRUE,
-				                                           'user'=> $key['pseudo'],
-				                                           'value'=> $key['email']]);
+				echo $this->getTwig()->render('profile.twig', ['space' => TRUE,
+				                                               'server' => $this->getServer(),
+				                                               'logged'=> TRUE,
+				                                               'user'=> $key['pseudo'],
+				                                               'admin' => $key['admin'],
+				                                               'value'=> $key['email']]);
 				exit();
 			}
 
 			if (strlen($value) < 4){
-				$noValid = true;
-				echo $this->_twig->render('profile.twig', ['noValid' => $noValid,
-				                                           'server' => $this->_server,
-				                                           'logged'=> TRUE,
-				                                           'user'=> $key['pseudo'],
-				                                           'value'=> $key['email']]);
+				echo $this->getTwig()->render('profile.twig', ['noValid' => TRUE,
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)